component/src/main/wso2/sap/producer.bal

```diff
@@ -62,78 +62,77 @@ public type Producer client object {
     # + CCMSInterfaceVersion - The version of the CCMS system-administration interface that expects
     #   the external product from the R/3 System
     # + return - Returns an error if sending the BAPI fails.
-    public remote extern function sendBapiMessage(xml content, boolean doTransaction, boolean doLogon,
+    public remote function sendBapiMessage(xml content, boolean doTransaction, boolean doLogon,
     string? productManufacturer = "", string? productName = "", string? interface = "",
-    string? CCMSInterfaceVersion = "") returns string|error;
+    string? CCMSInterfaceVersion = "") returns string|error = external;
 };
 
 # The Client endpoint configuration of SAP.
 #
 # + destinationName - Name of the SAP gateway
-# + sapClient - SAP Client
-# + userName - Username to log in to SAP
+# + ^"client" - SAP Client
+# + username - Username to log in to SAP
 # + password - Password to log in to SAP
 # + language - Language preferred by the user who is logged in
 # + sysnr - SAP system number
-# + aliasUser - Alias of the user who is logged in.
-# + mySapsso2 - The SAP cookie version 2 of the login ticket that should be used
+# + aliasuser - Alias of the user who is logged in.
+# + mysapsso2 - The SAP cookie version 2 of the login ticket that should be used
```

Comment:
Fix camel case here too.

Reply:
As per the Rajkumar, customer wanted this notation for sap parameters.